From 68e835953ac211f9e30b35fde3bee1d7aadfd40b Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 6 May 2016 14:21:21 +0800
Subject: [PATCH 1054/5242] MLK-12731-2 ARM: dts: Makefile: add dts entry for
 imx6q-arm2-hsic

commit  a3dcd4daef18ab374bdac3502319d31a2bc2b314 from
https://source.codeaurora.org/external/imx/linux-imx.git

This dts is only for USB HSIC controller test which needs
Validation Port Card on it.

Disable controller 3 due to strange signal on it at arm2 board.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile            |    1 +
 arch/arm/boot/dts/imx6q-arm2-hsic.dts |    8 --------
 2 files changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index ced452a..c4d77f1 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -442,6 +442,7 @@ dtb-$(CONFIG_SOC_IMX6Q) += \
 	imx6q-apalis-ixora-v1.1.dtb \
 	imx6q-apf6dev.dtb \
 	imx6q-arm2.dtb \
+	imx6q-arm2-hsic.dtb \
 	imx6q-b450v3.dtb \
 	imx6q-b650v3.dtb \
 	imx6q-b850v3.dtb \
diff --git a/arch/arm/boot/dts/imx6q-arm2-hsic.dts b/arch/arm/boot/dts/imx6q-arm2-hsic.dts
index d2cca3e..10c95ad 100644
--- a/arch/arm/boot/dts/imx6q-arm2-hsic.dts
+++ b/arch/arm/boot/dts/imx6q-arm2-hsic.dts
@@ -22,11 +22,3 @@
 	osc-clkgate-delay = <0x3>;
 	status = "okay";
 };
-
-&usbh3 {
-	pinctrl-names = "idle", "active";
-	pinctrl-0 = <&pinctrl_usbh3_1>;
-	pinctrl-1 = <&pinctrl_usbh3_2>;
-	osc-clkgate-delay = <0x3>;
-	status = "okay";
-};
-- 
1.7.9.5

